admin/listPurchases/listView.html

```diff
@@ -199,6 +199,21 @@
     }
 }
 
+function describeEntry(entry){
+    const preOrderTag = entry.isPreorder ? 'PRE:<br> ' : '';
+    const describedItems = entry.items
+        .map((item) => describeItem(item))
+        .join("<br>");
+    return `${preOrderTag}${describedItems}`;
+}
+
+function describeItem(item){
+    if(item.id && item.count){
+        return `${item.id}: <b>${item.count} stk</b>`;
+    }
+    return item.description;
+}
```

Comment:
Nice change!

*technically* it isn't an extraction because you are also changing behavior. In the future, split pure extractions (a refactoring) and logical changes (changes to behavior) into separate commits to make it easier to review.

The change to the logic is this: Preorders now have `PRE: ` on a separate line (`<br>` appended).